### ISSUE TITLE
Next New Moon, .......

### ISSUE THREAD

**Issue by Feboli** [solarissmoke/php-moon-phase#14]
Thanks for the good script.
However,  I verified this:
_Example_
Date selected: 02/01/2017 11:33:58 am.
Result:
-**next full moon: 14:54:49, 12 Mar 2017.**
-next new moon: 15:00:15, 26 Feb 2017....OK
-**next first quarter moon: 11:33:38, 5 Mar 2017.**
-**next last quarter moon: 16:01:29, 20 Mar 2017.**

But the right result should be:
- next full moon: 0:33:58, 11 Feb 2017.
- next first quarter moon: 4:19:47, 4 Feb 2017.
- next last quarter moon: 19:35:13, 18 Feb 2017.

**Comment by solarissmoke** [comment]
`full_moon()` returns the correct result for me. You must be using `next_full_moon()`. The difference between the two is that `full_moon()` will give you the time of the full moon in the current lunar cycle, and `next_full_moon()` will give you the full moon in the next lunar cycle (not the next full moon... bad function naming, I know).

Same goes for `next_first_quarter()` etc. I will update the documentation to make this more clear.

**Comment by Feboli** [comment]
You're right, sorry! ... unfortunately the function names have me confused.
It would be nice to have also other data such as:
- Latitude and longitude
- Moonrise and moonset

**Comment by solarissmoke** [comment]
It's unlikely that I will add these any time soon - I don't know enough about the algorithms for calculating them, and don't really work much with PHP these days. If someone wants to add this functionality then I'm more than happy to accept pull requests.

I will try and fix the documentation at some point with regard to your original issue.

**Comment by wolfgang42** [comment]
I agree that these function names are a tad confusing. Is the below algorithm correct for getting the date of the next full moon after the current date, or have I misunderstood how these functions work?

```php
if ($moon->full_moon() > $moon->timestamp) {
	// Full moon hasn't yet happened in current lunar cycle; next one is in this cycle
	return $moon->full_moon();
} else {
	// Full moon already passed in current lunar cycle; next one is in next cycle
	return $moon->next_full_moon();
}
```

**Comment by solarissmoke** [comment]
Original issue fixed in #34.
